client/src/styles/tools/mixins.scss

```diff
@@ -1,5 +1,5 @@
 @mixin truncate-with-ellipsis {
-  overflow: hidden;
+  overflow: visible;
```

Comment:
Several places use `@mixin truncate-with-ellipsis`. This change is likely to break functionality of those use cases. The `overflow: hidden` is a typical part of this CSS solution to truncation ([example reference](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/)).

Reply:
Returned overflow's value to hidden

Reply:
Did you need this change for your implementation? If so, do you still need it?

_I don't want to prevent a solution; I just want to verify whether the change is still necessary, should be isolated, or can be solved another way._

Reply:
No worries, I don't need it anymore, as the dropdown is no longer being cut off